rpc_jobs/unit/skip_build_check.yml

```diff
@@ -0,0 +1,76 @@
+- job:
+    name: RE-unit-test-skip-build-check
+    project-type: pipeline
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+    dsl: |
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.shared_slave{
+
+        env.RE_JOB_REPO_NAME = "test_repository"
+
+        stage("Setup repository") {
+          sh """#!/bin/bash -xeu
+            mkdir ${env.RE_JOB_REPO_NAME}
+            cd ${env.RE_JOB_REPO_NAME}
+            git init .
```

Comment:
Not a huge saving, but I think `common.shared_slave` already clones rpc-gating so you could probably just commit some dummy files there for testing?

Reply:
^^^ is probably a bad idea, prefer this.